modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -83,6 +80,17 @@ const questionnaireStyle = theme => ({
         top: 'auto',
         bottom: theme.spacing(1),
         right: theme.spacing(5)
+    },
+    newFormButtonWrapper: {
+        float: "right",
+        position: "inherit"
```

Comment:
You need to make this div a new positioned parent, so that the position of the loading indicator is relative to it. The right way to do that is to use `position: relative`. Inheriting the position coincidentally works because the card header happens to also be "relative", but that is a coincidence we shouldn't rely on.
```suggestion
        position: "relative"
```